Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -3282,15 +3282,6 @@ def get_remote_data_command(client: Client, params: Dict[str, Any], args: Dict)
     raw_context, context_version = get_integration_context_with_version()
     context_data = extract_context_data(raw_context.copy())
     events_limit = int(params.get('events_limit') or DEFAULT_EVENTS_LIMIT)
-    processed_offenses = print_mirror_events_stats(context_data, f"Starting Get Remote Data For "
```

Comment:
Any reason to completely remove the `print_mirror_events_stats` print here?

Reply:
Added back the `print_mirror_events_stats`